src/pages/SignUp/SignUp.tsx

```diff
@@ -35,89 +53,234 @@ const SignUp = () => {
         window.alert("error");
       });
   };
-  const handleChange = (e: any) => {
-    const { name, value } = e.target;
 
-    setSignUpFormData((prevFormData: any) => {
-      return { ...prevFormData, [name]: value };
-    });
-  };
   return (
     <div className="signup">
-      <h3>Sign Up</h3>
-      <div className="signup-box">
-        <div className="signup-form">
-          <form onSubmit={handleSubmit} autoComplete="false">
-            <div className="signup-form-element">
+      <h2>Sign Up</h2>
+      <div className="signup__box">
```

Comment:
ok, `signup` is your block and so is `signup-box`
same with `signup-form`
the consequent elements would be the labels etc
hence `signup-form__input-element-group`